build.gradle

```diff
@@ -34,6 +34,12 @@ plugins {
   id 'antlr'
 }
 
+allprojects {
+    group 'bio.terra'
```

Comment:
FYI - We need to change that to `bio.terra.datarepo` at some point.

Reply:
Are their larger ramifications in doing this? Shall I switch it in this PR? Also, should the datarepo-client group also be `bio.terra`?

Reply:
I will make a ticket for this